lib/consensus.js

```diff
@@ -974,10 +974,24 @@ function _tallyBranches({
           _computeMostRecentVotes({event, yElectorSet, yByElector});
         }
         // if some voting ancestors haven't computed support yet, defer
-        const votingEvents = Object.values(event._votes);
-        if(votingEvents.some(
-          e => e && e !== event && e._supporting === undefined)) {
-          next.push(event);
+        let pushed = false;
```

Comment:
We use a variable named `defer` in other places in the code for these situations.

```suggestion
        let defer = false;
```